### PR TITLE
Update move.yml

### PR DESCRIPTION
**Pull request by devoncarew**
@JekCharlsonYu @kevmoo @pq

### CODE REVIEW

File: .github/move.yml [hunk 1]
```diff
@@ -16,6 +16,6 @@ mentionAuthors: true
 keepContentMentions: true
 
 # Set custom aliases for targets
-# aliases:
-#   r: repo
-#   or: owner/repo
+aliases:
+  flutter: flutter/flutter
+  flutter: flutter/flutter-intellij
```
Comment (pq) [inline .github/move.yml]: I'm new to the syntax but it seems a little odd to have one alias for two targets.  Is that right?

Comment (devoncarew) [reply .github/move.yml]: No, good catch - the 2nd one should be `flutter-intellij`.

**Review by pq — APPROVED** [review]
👍

**Comment by devoncarew** [comment]
Hmm, I wonder how this gets merged through? I'd assumed that a companion CL would be created on gerrit.

**Comment by devoncarew** [comment]
hmm...

**Pull request closed without merging**